addon/components/ember-table-row.js

```diff
@@ -75,6 +77,15 @@ export default class EmberTableRow extends Component {
     return this.get('selectedRows').indexOf(this.get('rowValue')) >= 0;
   }
 
+  @computed('row.api.rowHeight')
```

Comment:
We should pass `estimateRowHeight` and `staticHeight` to the row and only apply the style if `staticHeight === true`